src/Events/Details/index.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Event from '../Event';
+import { getFormattedDate } from '../../core/Dates';
+import { searchConference } from './duck';
+
+class InsidePage extends Component {
+    static propTypes = {
+        event: PropTypes.shape({
+            data: PropTypes.instanceOf(Event),
+            error: PropTypes.number,
+            isFetching: PropTypes.bool,
+            lastFetched: PropTypes.number,
+        }),
+        alias: PropTypes.string.isRequired,
+        searchConference: PropTypes.func.isRequired,
+    };
+    static defaultProps = {
+        event: {},
+    };
+
+    componentDidMount() {
+        this.props.searchConference(this.props.alias);
+    }
+    /**
+     * Render speaker images
+     * @return {array}
+     */
+    renderSpeakers() {
+        const renderImages = [];
+        this.props.event.data.speakers.map((speaker) => {
+            renderImages.push(
+                <img
+                    className="rounded-circle mr-2"
+                    key={speaker.id}
+                    src={speaker.pictureUrl}
+                    width="40"
+                    height="40"
+                    alt={speaker.name}
+                />);
+        });
+        return renderImages;
+    }
+
+    /**
+     * Render tags
+     * @return {array}
+     */
+    renderTags() {
+        const renderTags = [];
+        this.props.event.data.tags.map((tag) => {
+            renderTags.push(<span
+                key={tag}
+                className="badge badge-pill badge-light mr-2"
+            >{tag}
+            </span>);
+        });
+        return renderTags;
+    }
+
+    /**
+     * Render description
+     * @return {object}
+     */
+    renderDescription() {
+        return { __html: this.props.event.data.description };
+    }
+
+    render() {
+        const { error, isFetching, data } = this.props.event;     
+
+        if (typeof isFetching === 'undefined') {
+            // nothing.
+            return 'Loading!';
+        }
+
+        if (isFetching) {
+            return (<p>Loading!</p>);
+        }
```

Comment:
Why the first loading is a plain text, but the second one is in a paragraph instead?